src/mds/MDLog.cc

```diff
@@ -147,18 +147,21 @@ uint64_t MDLog::get_safe_pos() const
   return journaler->get_write_safe_pos(); 
 }
 
-void MDLog::_log_estimated_complete_time(utime_t start_time, uint64_t start_pos){
+std::pair<double, uint64_t> MDLog::get_replay_estimated_complete_time(){
```

Comment:
Its good to return these encapsulated in a structure if its needs to be extended to add more information over time.